packages/wrangler/src/dev/local.tsx

```diff
@@ -247,6 +260,34 @@ function useLocalWorker({
 					logger.log("⎔ Reloading experimental local server.");
 					await current.setOptions(mf3Options);
 				}
+
+				try {
+					// fetch the inspector JSON response from the DevTools Inspector protocol
+					const inspectorJSONArr = (await (
+						await fetch(`http://127.0.0.1:${inspectorPort}/json`)
```

Comment:
FYI, this `fetch()` fails on Node 16, which doesn't have `fetch()` as a built-in yet. You'll have to import `node-fetch` or change the `package.json` to require Node 18.